pkg/cloud/affinity_groups_test.go

```diff
@@ -14,13 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package cloud_test
+package cloud
```

Comment:
No! We separate this on purpose.

cloud_test being a separate module helps us to detect code smell in un-testable regions.

It also helps us to pair down what libs actually make it into our final module.

Why was this change made?